### PR TITLE
doc: Fix typo in doc/live.md

### PR DESCRIPTION
**Pull request by kangtegong**
There's no opening bracket so I added it

Signed-off-by: kangtegong <tegongkang@gmail.com>

### CODE REVIEW

File: doc/uftrace-live.md [hunk 1]
```diff
@@ -631,7 +631,7 @@ With dynamic tracing, you can trace specific functions only given by the
 `-U`/`--unpatch` option.  With capstone disassembly engine you even don't need
 to (re)compile the target with the option above.  Now uftrace can analyze the
 instructions and (if possible) it can copy them to a different place and rewrite
-it to call `mcount()` function) so that it can be traced by uftrace.  After that
+it (to call `mcount()` function) so that it can be traced by uftrace.  After that
```
[the remark this answers is not in the document]
Comment (kangtegong) [reply doc/uftrace-live.md]: Okay, I'll amend the commit  right away!

**Comment by kangtegong** [comment]
Erased the closing bracket

**Comment by honggyukim** [comment]
Hmm.. what the name of this branch? It's shown as `unknown repository` and its link is broken.

**Comment by honggyukim** [comment]
It seems that this remote branch is somehow deleted and the state of this PR looks broken. Could you please send a new PR with the same change. I will close this.

**Pull request closed without merging**